### PR TITLE
fix mobile users affecting user ratio data

### PR DESCRIPTION
**Pull request by QvodSoldier**
#### What type of this PR
/kind bug

#### What this PR does / why we need it:
fix mobile application affecting user ratio data

#### Which issue(s) this PR fixes:
- [Erda Cloud Issue Link](https://terminus-org.app.terminus.io/erda/workBench/projects/387/issues/all?id=72814&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiMTAwMDQzNiJdfQ%3D%3D&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=429&type=BUG)

#### Specified Reviewers:

/assign @Effet

### CODE REVIEW

**Review by Effet — DISMISSED** [review]

File: modules/dicehub/service/publish_item/item_statistics.go [hunk 1]
```diff
@@ -1025,33 +1025,14 @@ func (i *PublishItem) EffactUsersRate(point uint64, start, end time.Time, av str
...
-		Apply("cardinality", "tags.uid").
-		LimitPoint(int(point))
-	if av != "" {
-		totalReq = totalReq.Filter("av", av)
-	}
-	totalResp, err := metrics.Client.QueryMetric(totalReq)
-	logrus.Infof("monitor request params 影响用户占比: %s", totalReq.ConstructParam().Encode())
 
+	// get cumulative users for each point
+	allUserResp, err := i.CumulativeUsers(point, start, end, mk)
```
[the remark this answers is not in the document]
Comment (QvodSoldier) [reply modules/dicehub/service/publish_item/item_statistics.go:1030]: The deleted code is a new user added at a time, this part requires the number of all users at a time

[the remark this answers is not in the document]
Comment (QvodSoldier) [reply modules/dicehub/service/publish_item/item_statistics.go:1030]: `CumulativeUsers` is an existing business interface, used to count all users at a certain moment

File: modules/dicehub/service/publish_item/item_statistics.go [hunk 2]
```diff
@@ -1025,33 +1025,14 @@ func (i *PublishItem) EffactUsersRate(point uint64, start, end time.Time, av str
...
-	totalDataList, err := parsingDataListResp(totalResp)
-	if err != nil {
-		return nil, err
-	}
-	allUsers := make([]uint64, 0, point)
-	for _, v := range totalDataList.Data.Results[0].Data[0] {
-		allUsers = v.Data
-	}
+	logrus.Debugf("all user resp: %v", allUserResp)
+	allUsers := allUserResp.Results[0].Data[0]["cardinality.tags.uid"].Data
```
[the remark this answers is not in the document]
Comment (QvodSoldier) [reply modules/dicehub/service/publish_item/item_statistics.go:1035]: `CumulativeUsers` will make sure there is data in it.

**Review by sfwn — DISMISSED** [review]

**Pull request merged**